React/package.json

```diff
@@ -41,5 +43,12 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "babel-eslint": "10.1.0",
+    "eslint": "7.32.0",
+    "eslint-config-devextreme": "0.1.40",
+    "eslint-plugin-no-only-tests": "2.6.0",
+    "eslint-plugin-react": "7.31.10"
```

Comment:
We added ```eslint-plugin-react``` but I don't see any usage of this package :)
Let's extend from it in the .eslintrc.js